docs/docs/inline-environments.mdx

```diff
@@ -0,0 +1,132 @@
+---
+name: "Inline environments"
+route: "/inline-environments"
+menu: Advanced features
+---
+
+# Inline environments
+
+Inline environments is the practice of defining the environment's config inline
+for evaluation at runtime as opposed to configuring it statically in
+`spec.json`.
+
```

Comment:
As a newcomer to the topic, I'd wish for a section here that gives a rough overview, along those lines:

* `spec.json` is no longer used
* The Jsonnet is expected to hold a `tanka.dev/Environment` object
* This object is expected to hold Kubernetes objects at `.data`

Reply:
PTAL, added what you said, simple and clear.